modules/ca/src/tools/tool_lib.c

```diff
@@ -639,3 +640,32 @@ int connect_pvs (pv* pvs, int nPvs)
     }
     return returncode;
 }
+
+
+/*+**************************************************************************
+ *
+ * Function:    use_ca_timeout_env
+ *
+ * Description: Set the variable at the given address to the
+ *              value of the CA timeout environment variable
+ *
+ * Arg(s) In:   timeout - Pointer to double
+ *
+ **************************************************************************-*/
```

Comment:
Can this comment be condensed and made less redundant?

eg. the function name and argument are syntactically unambiguous, and need not be repeated.

Also, why go through the extra indirection of an argument which will always point to the `caTimeout` global?

(I understand and agree with a general aversion to globals, though in this can that ship has already sailed)

Reply:
We don't need to maintain these decorated function header comments, feel free to change this to more like a single-line comment...